_sass/main.scss

```diff
@@ -0,0 +1,123 @@
+/* Global Styles */
+/* mixins, colors, etc. */
+
+$cfd-red: #E14E54;
+$navy: #181E34;
+$white: white;
+$light-grey: #C4C4C4;
+
+$breakpoints: (
+  'small':  ( min-width:  767px ),
+  'medium': ( min-width:  992px ),
+  'large':  ( min-width: 1200px )
+) !default;
+
+@mixin respond-to($breakpoint) {
+  // If the key exists in the map
+  @if map-has-key($breakpoints, $breakpoint) {
+    // Prints a media query based on the value
+    @media #{inspect(map-get($breakpoints, $breakpoint))} {
+      @content;
+    }
+  }
+ 
+  // If the key doesn't exist in the map
+  @else {
+    @warn "Unfortunately, no value could be retrieved from `#{$breakpoint}`. "
+        + "Available breakpoints are: #{map-keys($breakpoints)}.";
+  }
+}
+
+/* Structural Responsive Needs */
+
+html, body {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+}
+
+/* typeography */
+
+h1 {
+  font-family: Montserrat;
```

Comment:
Do we need to specify 'Monsterrat' here if this is the default font-family for all elements in html, body? Same goes for the other elements below.

Reply:
No, we do not need to re-specify the font family. We will need to clean up the css as we go. I made some comments on the h1 styles that will need to be updated too.